fire-cracker/SendIT-react
# #165118209 Users should be able to Cancel Order

#### What does this PR do?
- Implements user can cancel order

#### How should this be manually tested?
- Clone repo
- Check out to branch `feature/165118209/cancel-order`
- Run application with `npm run start-dev` in the terminal

#### What are the relevant pivotal tracker stories?
- [Users should be able to Cancel Order (react)](https://www.pivotaltracker.com/story/show/165118209)
